kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/kms/vault/KubeVaultTestKmsFacade.java

```diff
@@ -185,7 +160,7 @@ public void generateKek(String alias) {
 
         @Override
         public void deleteKek(String alias) {
-            throw new UnsupportedOperationException();
+            throw new UnsupportedOperationException("KEK deletion is not supported in Vault");
```

Comment:
I know you've just fixed the message, but is throwing an exception here the right option? It means the interface is a leaky abstraction and all callers can do is log the exception or check if the instance they have been given is vault and avoid calling it. I wonder if we should just log at `error` or `warn` here and do nothing about delete requests to vault? 

cc: @k-wall

Reply:
Agree with logging a `warn` here and do nothing else